### ISSUE TITLE
Migration 0008_changed_logo_fields.py failed if no logo set

### ISSUE THREAD

**Issue by emanuelschuetze** [OpenSlides/OpenSlides#3782]
Try to migrate a database (without any logo configuration) you get this python error:
```
...0008_changed_logo_fields.py", line 48, in move_old_logo_settings
    logo_pdf_L.value['path'] = logo_pdf.value.get('path', '')
TypeError: 'str' object does not support item assignment
```
@FinnStutzenstein Any hotfix?

**Comment by ostcar** [comment]
https://github.com/OpenSlides/OpenSlides/pull/3786

This is the line, that fixes the issue: https://github.com/OpenSlides/OpenSlides/pull/3786/files#diff-a3f4720a9aaa2c83c3211c5205938c36R45

**Comment by emanuelschuetze** [comment]
Closed, becuase it is fixed in #3786
